csharp/test/Ice/proxy/MyDerivedClassAMDI.cs

```diff
@@ -6,7 +6,7 @@ namespace proxy
     {
         namespace AMD
         {
-            public sealed class MyDerivedClassI : Test.MyDerivedClassDisp_
+            public sealed class MyDerivedClassI : Test.MyDerivedClassDisp_, Ice.Object
```

Comment:
You need to implement directly the interface you're "overriding".

In practice, it's unlikely anybody overrides these operations. But at least this test shows it's doable.